lib/operations/logic/falsy.js

```diff
@@ -0,0 +1,8 @@
+import truthy from '../../helpers/truthy';
+
+function falsy(a) {
+  return !truthy(a);
+}
+
+falsy.code = '!';
```

Comment:
Same here `code` as described in https://github.com/axa-ch/json-logic-js/pull/1/files#r259730014

`!` is an existing operator